src/ucp/amo/nb_amo.c

```diff
@@ -13,24 +13,191 @@
 #include <ucs/debug/log.h>
 #include <inttypes.h>
 
+#define UCP_UCT_AMO_OPCODE_POST(_opcode, _op)                 \
+    if (ucs_unlikely((_opcode) >= UCP_ATOMIC_POST_OP_LAST)) { \
```

Comment:
need to also condition on ENABLE_PARAMS_CHECK to avoid this if in release mode

Reply:
updated, these macro were used only once - expanded into functions